src/obsidian_ai_hub/coding/backend.py

```
@@ -0,0 +1,317 @@
+        )
+
+        stdout_chunks: list[str] = []
+        stderr_chunks: list[str] = []
+        cancelled = False
+
+        def read_pipe(pipe, container):
+            try:
+                for line in iter(pipe.readline, ""):
+                    container.append(line)
+            except Exception:
+                pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)